app/Domain/Clients/Templates/newClient.blade.php

```diff
@@ -38,7 +38,7 @@
                                     inputType="text"
                                     name="name"
                                     id="name"
-                                    value="{{ $tpl->escape($values['name']) }}"
+                                    value="{{ $tpl->escape($values->name) }}"
```

Comment:
{{ }} is already escaping so you can just use the value directly {{ $values->name }}